pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -239,14 +239,14 @@ def __new__(cls, implementation='binary_heap', **kwargs):
         comp = kwargs.get("comp", lambda u, v: u < v)
         if implementation == 'linked_list':
             return LinkedListPriorityQueue(comp)
-        elif implementation == 'binary_heap':
+        if implementation == 'binary_heap':
             return BinaryHeapPriorityQueue(comp)
-        elif implementation == 'binomial_heap':
+        if implementation == 'binomial_heap':
             return BinomialHeapPriorityQueue()
-        else:
-            raise NotImplementedError(
-                "%s implementation is not currently supported "
-                "by priority queue.")
```

Comment:
We need these bunch of `if-elif` for better readability. Please restore the original versions here.

Reply:
But if `if` is calling `return` then removing `if-else` is better approach I guess.

Reply:
> better readability